.github/workflows/integration.yml

```diff
@@ -34,10 +36,31 @@ on:
   # manual trigger
   workflow_dispatch:
     inputs:
-      dbt-core-branch:
-        description: "branch of dbt-core to use in dev-requirements.txt"
+      name:
+        description: "Name to associate with run (example: 'dbt-adapters-242')"
         required: false
         type: string
+        default: "Adapter Integration Tests"
+      adapter_branch:
+        description: "The branch of this adapter repository to use"
+        type: string
+        required: false
+        default: main
```

Comment:
Minor nit, but the missing double quotes is making my OCD twitch. Alternatively, remove double quotes from the other string values, which appears to be the style in this particular file (e.g. all of the existing name fields don't use quotes).

Reply:
Fixed.